We strip binaries downstream to aid in the generation of backtraces.
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Backtraces#Stripping
https://github.com/greshake/i3status-rust/issues/1863
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -104,5 +104,5 @@ features = ["json"]
 
 [profile.release]
 lto = "thin"
-strip = true
+strip = false
 debug = 1
